roles/inspec/defaults/main.yml

```diff
@@ -27,7 +27,7 @@ inspec:
       skip_controls: []
       attributes:
         RHEL_07_030710_audit_key_name: audit_account_changes-V-38531
-        client_alive_interval: '900'
+        client_alive_interval: '1800'
```

Comment:
Where is ursula we setting this value? If this is tunable setting than it should be pointing to that variable and can default to 1800.